CONTRIBUTING.md

```diff
@@ -106,6 +106,10 @@ If after running these the CI flow is still failing, try updating `flake8`,
 `isort` and `black`. This can be done by running `pip install --upgrade black`,
 `pip install --upgrade flake8`, and `pip install --upgrade isort`.
 
+If you face issues while running the shell scripts and face errors like 
+`: invalid option` one possible reason might be that the line endings are 
```

Comment:
This looks good, but can you post the full error in a comment here or on this issue so we can take a look?